File: src/main/java/org/team1/nbe1_3_team01/domain/user/entity/Role.java
package org.team1.nbe1_3_team01.domain.user.entity;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum Role {
    ADMIN("ROLE_ADMIN"),
    USER("ROLE_USER");
    private final String key;
}

File: src/main/java/org/team1/nbe1_3_team01/domain/user/entity/Role.kt
package org.team1.nbe1_3_team01.domain.user.entity

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class Role(
    val key: String
) {
    ADMIN("ROLE_ADMIN"),
    USER("ROLE_USER");
}
